.changeset/stupid-eagles-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-code-coverage': patch
+---
+
+Updated layout to improve contrasts and consistency with other plugins
```

Comment:
that's not the only change, would it be a lot to ask you to split it into two PRs? One for visuals and one to fix the bug?

Reply:
You are right, that makes a lot of sense! I'll look into that!

Reply:
Okay @alde, fixed now! I'll create a new PR with the fix once this PR has been reviewed and merged. _Note that print screens are not updated._